metabase/metabase
# make compact formatting consistent across all series

### Description

Previously, when data labels formatting was set to "auto", we for each individual series computed whether its labels should be compact or not. However, we [decided](https://metaboat.slack.com/archives/C064QMXEV9N/p1717113341767579) to go with consistency and now we propagate compactness to all series if it is sensible for at least one series. This applies to stack total labels as well: if all series auto formatting is full but totals should be compact, we render all labels as compact ones.

### How to verify

- check visual specs variants
- try different combinations of multiseries charts where compactness propagates to all series

### Demo

<img width="573" alt="Screenshot 2024-06-05 at 1 20 17 AM" src="https://github.com/metabase/metabase/assets/14301985/438e39c3-6c2a-404c-819d-5880717e1fea">

### Checklist
